apps/trading-e2e/src/integration/home.cy.ts

```diff
@@ -81,20 +81,12 @@ const generateProposal = (code: string): ProposalListFieldsFragment => ({
             __typename: 'DataSourceDefinition',
             sourceType: {
               __typename: 'DataSourceDefinitionInternal',
-              sourceType: {
```

Comment:
Internal termination is a case only for termination and not for settlement. So this part should not go back in when the time comes.
Same goes for the proposal definitions

Reply:
awesome! thanks for the clarification!